packages/backend-app-api/src/wiring/ServiceRegistry.test.ts

```diff
@@ -125,6 +125,7 @@ describe('ServiceRegistry', () => {
 
   it('should not be possible for root scoped services to depend on plugin scoped services', async () => {
     const factory = createServiceFactory({
+      // @ts-expect-error
```

Comment:
It's worth pointing out that the errors emitted by TS aren't super nice. Since createServiceFactory has so many alternatives, it gets a bit confused, and it doesn't have a squiggly on `pluginDep` below but rather a quite deep one on on the service ref.